resources/run_adl.sh

```diff
@@ -224,6 +226,7 @@ if [[ $? -ne 0 ]]; then
     STORES='{ "name" : "localmongo", "provider" : "mongodb", "uri" : "mongodb://localhost:%s" }'
     STORES=$(printf "$STORES" "${MONGOD_PORT}")
     DATABASES=$(cat $DB_CONFIG_PATH)
+    TENANT_CONFIG="$MONGOHOUSE_DIR/testdata/config/mongodb_local/tenant-config.json"
```

Comment:
This TENANT_CONFIG variable is also used in the else statement, line 270.  
The logic to set the variable will either need to be added there.